docs/command-reference.md

```diff
@@ -486,17 +520,217 @@ Arguments:
+```
+
+The `show balance` displays the balance of a single NEP-17 asset (including NEO and GAS) of a specific account.
+
+### neoxp show balances
+
+```
+Usage: neoxp show balances [options] <Account>
+
+Arguments:
+  Account             Account to show asset balances for
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show balances` displays the balance of all NEP-17 asset (including NEO and GAS) owned by a specific account.
+
+### neoxp show block
+
+```
+Usage: neoxp show block [options] <BlockHash>
+
+Arguments:
+  BlockHash           Optional block hash or index. Show most recent block if
+                      unspecified
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show block` displays the contents of a single block, specified by index or hash.
+
+### neoxp show nft
+
+```
+Usage: neoxp show nft [options] <Contract> <Account>
+
+Arguments:
+  Contract            NFT Contract (Symbol or Script Hash)
+  Account             Account to show NFT (Format: Script Hash, Address, Wallet name)
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show nft` displays the content of an NFT contract for a specified account.
+
+### neoxp show notifications
 
-- `show balance` will display the balance of a single NEP-17 asset (including NEO and GAS) of a specific account
-- `show balances` will display the balance of all NEP-17 asset (including NEO and GAS) owned by a specific account
-- `show block` with display the contents of a single block, specified by index or hash
-- `show transaction` with display the contents of a transaction specified by hash and its execution results if available
-  - `show tx` is an alias for `show transaction`
+```
+Usage: neoxp show notifications [options]
+
+Options:
+  -c|--contract <CONTRACT>      Limit shown notifications to the specified contract
+  -n|--count                    Limit number of shown notifications
+  -e|--event-name <EVENT_NAME>  Limit shown notifications to specified event name
+  -i|--input <INPUT>            Path to neo-express data file
+```
+
+The `show notifications` displays contract notifications in JSON format.
+
+### neoxp show transaction
+
+```
+Usage: neoxp show transaction [options] <TransactionHash>
+
+Arguments:
+  TransactionHash     Transaction hash
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show transaction` displays the contents of a transaction specified by hash and its execution results if available.
+
+`show tx` is an alias for `show transaction`
+
+## neoxp candidate
+
+The `candidate` command has a series of subcommands for managing candidates election in the Neo-Express blockchain.
+
+### neoxp candidate list
+
+```
+Usage: neoxp candidate list [options]
+
+Options:
+  -t|--trace          Enable contract execution tracing
+  -j|--json           Output as JSON
+  -i|--input <INPUT>  Path to neo-express data file
+```
```

Comment:
The results include the public key of the candidate and the number of ballots cast.